test/after.js

```diff
@@ -10,12 +10,14 @@ import { WebDriverHelper as helper } from '../src/index';
 
 const afterHookTimeoutMs = 30000;
 
-test.afterEach( 'Take screenshot', function() {
```

Comment:
I'm not against it or disagreeing, but curious why description on the hook is removed?

It could be useful to to pinpoint errors once the hooks are growing.

```
  WebDriverHelper
    ✓ has function "waitTillPresentAndDisplayed" to wait until element is present and displayed (136ms)
    1) "after each" hook: Take screenshot for "has function "waitTillPresentAndDisplayed" to wait until element is present and displayed"


  1 passing (4s)
  1 failing

  1)  "after each" hook: Take screenshot for "has function "waitTillPresentAndDisplayed" to wait until element is present and displayed":
     Error: fsdfs
```

Reply:
@gedex, it seems that the newer version of selenium no longer supports passing a description. I don't know more about this change. To create this PR, I basically followed the changes done to update `selenium-webdriver` on `wp-e2e-tests`. See https://github.com/Automattic/wp-e2e-tests/pull/429. Specifically these two comments: https://github.com/Automattic/wp-e2e-tests/pull/429#issuecomment-286766674 and https://github.com/Automattic/wp-e2e-tests/pull/429#issuecomment-286827231.

Reply:
Ah, thanks for the explanation. I don't know about the discrepancies between latest selenium-webdriver and mocha, but it seems we can follow the same path here.